test/e2e/tests/single_zone_pd_extreme_e2e_test.go

```diff
@@ -272,4 +272,64 @@ var _ = Describe("GCE PD CSI Driver pd-extreme", func() {
 			Expect(gce.IsGCEError(err, "notFound")).To(BeTrue(), "Expected disk to not be found")
 		}()
 	})
+
+	It("Should complete entire disk lifecycle with underspecified volume ID", func() {
```

Comment:
@sunnylovestiramisu How many test cases can be shared between extreme pd and other pd types where the only difference is the type of disk created?

It may be worth considering refactoring the test cases such that you can share the same test logic and just swap out the disk type.

Reply:
Happy to do this. It started as a small subset, but seeing as this test case is compatible with pd_extreme, there may be others.

We should put this back into a single file and use [table-driven tests](https://pkg.go.dev/github.com/onsi/ginkgo@v1.16.5/extensions/table).

Reply:
I filed #1117 to take care of this. I had issues with the ginkgo dependency, so I will do it in a separate PR. I'm considering upgrading to ginkgo 2.X